tests/test_issue_5164_starship_prompt_composition.py

```
@@ -266,7 +267,45 @@ def test_plain_bash_bootstrap_installs_cmux_prompt_command() -> None:
+            "CMUX_SOCKET_PATH": "",
+            "CMUX_TAB_ID": "tab-test",
+        }
+    )
+    script = r"""
+source "$CMUX_BASH_INTEGRATION"
+downstream_prompt_hook() { printf 'STATUS=%s\n' "$?"; }
+(exit 7)
+_cmux_prompt_command
+downstream_prompt_hook
+"""
+    proc = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)